sdk/provisioning/Azure.Provisioning/tests/ProvisioningTests.cs

```diff
@@ -127,6 +131,35 @@ public async Task WebSiteUsingL3()
 
             infra.GetSingleResource<ResourceGroup>()!.Properties.Tags.Add("key", "value");
             infra.GetSingleResourceInScope<KeyVault>()!.Properties.Tags.Add("key", "value");
+
+            foreach (var website in infra.GetResources().Where(r => r is WebSite))
```

Comment:
Filed https://github.com/Azure/azure-sdk-for-net/issues/42178 to consider adding a generic method for multiple constructs/resources.